pyproject.toml

```diff
@@ -0,0 +1,45 @@
+[build-system]
+requires = [
+  "setuptools>=42",
+  "setuptools_scm[toml]>=6.2",
+]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools_scm]
+
+[project]
+name = "peek_data"
+description = "Live Data Reduction Dashboard"
+license = { file = "LICENSE" }
+readme = "README.md"
+classifiers = [
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: BSD License",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+]
+requires-python = ">=3.9"
```

Comment:
Can this be lowered to 3.8? Mantid still requires 3.8 for at least half a year. And we need Mantid to load most non-ESS files.

Reply:
Are those non-ESS files needed for the data reduction...?

Reply:
Ultimately, no. But for testing current workflows, yes. E.g. all files for SANS and powder diffraction that we currently have need to be loaded with Mantid.